docs/remove_controller.md

```diff
@@ -0,0 +1,32 @@
+# Remove or replace a controller
+
+You can manually remove or replace a controller from a multi-node k0s cluster (>=3 controllers) without downtime.
+
+## Remove a controller
+
+You first have to delete the controller from Kubernetes itself.
+To do so, run the following commands from the controller:
+
+```shell
+k0s kubectl drain --ignore-daemonsets --delete-emptydir-data <controller>
+k0s kubectl delete node <controller>
+```
```

Comment:
These steps are needed only in the case the controller is running also as a worker with `--enable-worker`